app/models/proof_attempt.rb

```diff
@@ -1,13 +1,35 @@
 class ProofAttempt < ActiveRecord::Base
+  include Numbering
+
+  numbering_parent_column 'sentence_id'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.